ibc-apps/ics721-nft-transfer/types/src/data.rs

```diff
@@ -86,6 +91,7 @@ impl<'de> serde::Deserialize<'de> for Data {
 #[derive(Clone, Debug, Default, PartialEq, Eq)]
 pub struct Ics721Data(BTreeMap<String, DataValue>);
 
+#[cfg(feature = "serde")]
 impl FromStr for Ics721Data {
```

Comment:
I find this interesting as this general `FromStr` trait is only implemented with `serde`. I would expect that with `borsh` or any other similar features, this is implemented - or this is necessary?